A/AWSCRT/bundled/patches/aws-c-common.patch

```diff
@@ -0,0 +1,151 @@
+diff --git a/include/aws/testing/aws_test_harness.h b/include/aws/testing/aws_test_harness.h
+index 68e3ade..4b84f7e 100644
+--- a/include/aws/testing/aws_test_harness.h
++++ b/include/aws/testing/aws_test_harness.h
+@@ -459,7 +459,11 @@ static inline int s_aws_run_test_case(struct aws_test_harness *harness) {
+ /* https://docs.microsoft.com/en-us/windows/console/console-virtual-terminal-sequences */
+ #ifdef _WIN32
+ 
++#if defined(__MINGW32__) || defined(__MINGW64__)
++#    include <windows.h>
++#else
+ #    include <Windows.h>
++#endif
```

Comment:
This should really just be
```c
include <windows.h>
```

Reply:
That will also work on `msbuild.exe`?

Reply:
See for example https://en.wikipedia.org/wiki/Windows.h, and also https://docs.microsoft.com/en-us/windows/win32/learnwin32/your-first-windows-program (although Microsoft docs isn't always consistent about casing, they just don't care at all about casing)